pkg/cli/run_test.go

```diff
@@ -22,3 +27,100 @@ func TestRunArgs_Env(t *testing.T) {
 	assert.Equal(t, "y", opts.Env[1].Name)
 	assert.Equal(t, "1", opts.Env[1].Value)
 }
+
+func TestRunMemory(t *testing.T) {
+	type fields struct {
+		All   bool
+		Type  []string
+		Force bool
+	}
+	type args struct {
+		cmd    *cobra.Command
+		args   []string
+		client *testdata.MockClient
+	}
+	var _, w, _ = os.Pipe()
+	tests := []struct {
+		name           string
+		fields         fields
+		args           args
+		wantErr        bool
+		wantOut        string
+		commandContext CommandContext
+	}{
+		{
+			name: "acorn run -h", fields: fields{
+				All:   false,
+				Type:  nil,
+				Force: true,
+			},
+			commandContext: CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader(""),
+			},
+			args: args{
+				args:   []string{"-h"},
+				client: &testdata.MockClient{},
+			},
+			wantErr: false,
+			wantOut: "./testdata/run/acorn_run_help.txt",
+		},
+		{
+			name: "acorn run -m found.container=256Miii found ", fields: fields{
+				All:   false,
+				Type:  nil,
+				Force: true,
+			},
+			commandContext: CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader(""),
+			},
+			args: args{
+				args:   []string{"-m found.container=256Miii", "found"},
+				client: &testdata.MockClient{},
+			},
+			wantErr: true,
+			wantOut: "invalid number \"256Miii\"",
+		},
+		{
+			name: "acorn run -m found.container=notallowed found ", fields: fields{
+				All:   false,
+				Type:  nil,
+				Force: true,
+			},
+			commandContext: CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader(""),
+			},
+			args: args{
+				args:   []string{"-m found.container=notallowed", "found"},
+				client: &testdata.MockClient{},
+			},
+			wantErr: true,
+			wantOut: "illegal number start \"notallowed\"",
+		},
+	}
+	for _, tt := range tests {
+		r, w, _ := os.Pipe()
```

Comment:
The for body should be wrapped with the below snippet so that tt.name is passed to the go test runner and the test name shows up.  Additionally tools will allow you to run just that one subtest easier (goland does this).
```
t.Run(tt.name, func(t *testing.T) {
})
```